markdown/the_photomode_wishlist.md

```diff
@@ -3,74 +3,79 @@
+- Photomode availability during cutscenes
+  : We are aware of how many tricks devs pull outside of the camera range in cutscenes, but being able to still use the photomode during them is crucial to have the most amount of variety when it comes to shooting scenes. Besides, players want to show how pretty your game is as much as you do, they won't focus on these imperfections happening around the scene.
 
-- Camera Tilt
-: It is often overlooked, but part of the complete movement of a free camera is being able to tilt it (also called roll depending on the bibliography). Such movement can help a lot in making compositions feel more dynamic or allow the users to shoot a vertical aspect ratio (although we would still very much prefer being able to [hotsample to the desired AR](basics.htm#composing-for-custom-aspect-ratios)).
+- Camera tilt
+  : It is often overlooked, but part of the complete movement of a free camera is being able to tilt it (also called roll depending on the bibliography). Such movement can help a lot in making compositions feel more dynamic or allow players to shoot in a vertical aspect ratio (although being able to [hotsample to the desired aspect ratio](basics.htm#composing-for-custom-aspect-ratios) is much preferred).
 
 - No camera clipping
-: Having the camera be free to move through geometry can be very useful, as it allows the user to position the camera in those places where it makes the most sense for the shot instead of only in the areas where the photomode allows them to.
+  : Having the camera be free to move through geometry can be very useful, as it allows the player to position the camera in those places where it makes the most sense for the shot instead of only in the areas where the photomode allows them to.
+
+- Photomode settings toggle
+  : Pretty obvious considering we want to shoot the game itself without any menu getting in the way. Even if your game has a feature that automatically outputs the screenshot into a file, giving the player the option to take a shot of the game with their preferred software would also be a nice perk. Additionally, avoid rendering any kind of watermark in the photomode; they get in the way and end up being very distracting to the overall photo.
+
+### Field-of-view
+
+Being able to control the "angle of the lens" of the camera is very important, but sometimes these have annoying quirks. Try to allow the minimum and maximum field of view that the engine supports, and provide enough granularity in the user interface so the player can choose the right FoV for their shot. Consider the real-world analogue to this control too: smaller values should give a wider view, larger values should be more narrow.
```

Comment:
Do you mean that you want the values on the pm to mimic a real lens instead of the fov values? I am not sure if I am into that. I understand you want to draw a 1:1 with photography, but I believe it just adds an unnecessary layer of abstraction.

Reply:
You mean the last sentence there? It's not so much about having the FoV values match to lens millimeter numbers or anything, but for players with experience with cameras it wouldd be less confusing for to move an FoV slider down to give a wider view and up to get a narrow view.

Reply:
I personally believe that there is a higher chance of players being more familiar with how FoV values work because of settings on FPS games than players that have ever used an irl camera in their lives. That plus the fact that I am more familiar with game engines than the average player means I am clearly biased, but that's how I see it.

Reply:
Ah I didn't realise that was a common setting in FPS games — we can just drop that mention completely then.